### ISSUE TITLE
Allow Elvui to work with GSE: Sequences, Variables, Macros Addon

### ISSUE THREAD

**Issue by justinltodd** [tukui-org/ElvUI#1353]
### Troubleshooting Steps

Does not allow keybind or utilzing the action bar 
https://www.curseforge.com/wow/addons/gse-gnome-sequencer-enhanced-advanced-macros

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [X] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by 1Botanica1** [comment]
That would be for GSE to be set up to work with ElvUI

I can see they have a ticket for it https://github.com/TimothyLuke/GSE-Advanced-Macro-Compiler/issues/1628

**Comment by TimothyLuke** [comment]
In order to do this GSE registers a custom action onto the "1" state of LibActionButton.  This allows for the ActionButton to passthrough directly to the GSE Button and mimics the Macro Feedback functions from before TWW.  (1 min demo: https://github.com/user-attachments/assets/fe885f67-b99a-4e77-9fdc-44960789b407)  This specific bind functionality is specific to ElvUI but has one issue that I cannot resolve.

I achieve the bind via 
```
_G[Button]:SetState(
   "1",
   "custom",
   {
       func = function(self)
           self:SetAttribute("type", "click")
           self:SetAttribute("clickbutton", _G[OtherButton])
        end,
        tooltip = "GSE: " .. OtherButton,
        texture = "Interface\\Addons\\GSE_GUI\\Assets\\GSE_Logo_Dark_512.blp"
    })
    ```
    where Button is ElvUI_BarNButtonM and OtherButton is a GSE Sequence.  This is working for all cases except if you dismount in combat taint prevention blocks the SetAttribute commands from func().  From pulling apart LibActionButton and Elv's code calling it, I haven't been able to identify a way to securely set this.  I am assuming that it needs something special on the LibActionButton end.
    
As a short term fix I have changed func to be 
```
                    func = function(self)
                        if not InCombatLockdown() then
                            self:SetAttribute("type", "click")
                            self:SetAttribute("clickbutton", _G[Sequence])
                        end
                    end,
```                    
But any insight would be greatly appreciated.

**Comment by TimothyLuke** [comment]
This function from GSE's end was shipped in GSE 3.2.14 so this ticket could be closed.
